### PR TITLE
[#1] [Chore] Setup KMM project

### PR DESCRIPTION
**Pull request by markgravity**
https://github.com/markgravity/kmm-ic/issues/1

## What happened 👀

- Initialize iOS Project based on the iOS Template
- Create a KMM Library project and add it to iOS Project as a Cocoapod lib
- Remove redundant frameworks: RxSwift, RxCocoa...
- Migrate from UIKit to SwiftUI
- Update CODEOWNER

## Insight 📝

There are no CI/CD runs at the moment, we will set it up at #47 

## Proof Of Work 📹

CleanShot 2023-04-11 at 09 50 49

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+#Kotlin
+kotlin.code.style=official
+
+#Android
+android.useAndroidX=true
+android.nonTransitiveRClass=true
+
+#MPP
+kotlin.mpp.enableCInteropCommonization=true
```
Comment (suho) [inline gradle.properties]: Please add new line at the end of the file

File: ios/.github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# @markgravity is the Team Lead and the others are team member
```
Comment (suho) [inline ios/.github/CODEOWNERS]: Please remove this file

File: ios/.github/ISSUE_TEMPLATE.md
Comment (suho) [inline ios/.github/ISSUE_TEMPLATE.md:1]: Please remove the folder `.github` here

File: ios/.gitignore
Comment (suho) [inline ios/.gitignore:1]: Same here, please remove this file

File: ios/KMMIC/Sources/Presentation/Modules/HomeViewController.swift
Comment (suho) [inline ios/KMMIC/Sources/Presentation/Modules/HomeViewController.swift:1]: You should remove this because you are using SwiftUI

File: ios/LICENSE
Comment (suho) [inline ios/LICENSE:1]: Same here, please remove this

File: ios/bitrise.yml
Comment (suho) [inline ios/bitrise.yml:1]: Same here! Please remove all unnecessary files

**Review by suho — CHANGES_REQUESTED** [review]

File: ios/bitrise.yml
Comment (markgravity) [reply ios/bitrise.yml:1]: Updated at [f1f75f9](https://github.com/markgravity/kmm-ic/pull/50/commits/f1f75f96fa42793d71173dd2be60628857c4928a)

File: ios/LICENSE
Comment (markgravity) [reply ios/LICENSE:1]: Updated at [f1f75f9](https://github.com/markgravity/kmm-ic/pull/50/commits/f1f75f96fa42793d71173dd2be60628857c4928a)

File: ios/KMMIC/Sources/Presentation/Modules/HomeViewController.swift
Comment (markgravity) [reply ios/KMMIC/Sources/Presentation/Modules/HomeViewController.swift:1]: Updated at [f1f75f9](https://github.com/markgravity/kmm-ic/pull/50/commits/f1f75f96fa42793d71173dd2be60628857c4928a)

File: ios/.gitignore
Comment (markgravity) [reply ios/.gitignore:1]: Updated at [f1f75f9](https://github.com/markgravity/kmm-ic/pull/50/commits/f1f75f96fa42793d71173dd2be60628857c4928a)

File: ios/.github/ISSUE_TEMPLATE.md
Comment (markgravity) [reply ios/.github/ISSUE_TEMPLATE.md:1]: Updated at [f1f75f9](https://github.com/markgravity/kmm-ic/pull/50/commits/f1f75f96fa42793d71173dd2be60628857c4928a)

File: ios/.github/CODEOWNERS (hunk 2 above)
Comment (markgravity) [reply ios/.github/CODEOWNERS]: Updated at [f1f75f9](https://github.com/markgravity/kmm-ic/pull/50/commits/f1f75f96fa42793d71173dd2be60628857c4928a)

File: gradle.properties (hunk 1 above)
Comment (markgravity) [reply gradle.properties]: Updated at [f1f75f9](https://github.com/markgravity/kmm-ic/pull/50/commits/f1f75f96fa42793d71173dd2be60628857c4928a)

**Review by suho — APPROVED** [review]

**Review by phongvhd93 — APPROVED** [review]

**Pull request merged**